### ISSUE TITLE
Manual styles don't work in media blocks

### ISSUE THREAD

**Issue by aeriksson** [Jarzka/stylefy#65]
It seems that manual styles in media blocks are ignored.

For instance, the code
```clojure
{::sf/media {{:min-width :0px} {::sf/manual [[" *" {:border :solid}]]}}}
```
should produce something like:
```
@media (min-width: 0px) {
  .prefix_291268972 * {
    border: solid
  }
}
```

However, it produces the following:
```css
@media (min-width: 0px) {

  .prefix_291268972 {

  }

}
```

**Comment by Jarzka** [comment]
This is true, `::stylefy/manual` is not supported inside `::stylefy/media` at the moment.

A workaround is to use `::stylefy/manual` at the top level and create media queries with `at-media` Garden helper function (there are some examples of this in the README file).
